templates/frontend/objects/article_details.tpl

```diff
@@ -133,13 +133,17 @@
 
 		<div class="entry_details">
 
-			{* Issue cover image *}
-			{if $issue->getLocalizedCoverImage()}
+			{* Article/Issue cover image *}
+			{if $article->getLocalizedCoverImage() || $issue->getLocalizedCoverImage()}
 				<div class="item cover_image">
 					<div class="sub_item">
-						<a href="{url page="issue" op="view" path=$issue->getBestIssueId($currentJournal)}">
-							<img src="{$publicFilesDir}/{$issue->getLocalizedCoverImage()|escape}">
-						</a>
+						{if $article->getLocalizedCoverImage()}
+							<img src="{$publicFilesDir}/{$article->getLocalizedCoverImage()|escape}"{if $article->getLocalizedCoverImageAltText()} alt="{$article->getLocalizedCoverImageAltText()|escape}{/if}">
+						{else}
+							<a href="{url page="issue" op="view" path=$issue->getBestIssueId($currentJournal)}">
```

Comment:
getBest...Id functions do not have any parameter any more -- apparently I have forgotten to change it here :-\ Could you please? Or I can do it after it is merged...